src/components/SchemaRow/TypeSelector.tsx

```diff
@@ -0,0 +1,24 @@
+import { Select } from '@stoplight/mosaic';
+import * as React from 'react';
+
+import type { useChoices } from './useChoices';
+
+type TypeSelectorProps = ReturnType<typeof useChoices>;
+
+export const TypeSelector = ({ choices, setSelectedChoice, selectedChoice }: TypeSelectorProps) => {
```

Comment:
What's the point of this component? It is not used anywhere here from what I can see.

Reply:
@marcelltoth some info on this? @mallachari is right, it doesn't seem to be used at all.

Reply:
Oh, right. That was an attempt to extract the common selector from `SchemaRow` and `TopLevelSchemaRow` but it turned out that the two are different so it didn't make sense to do it in the end. I removed it now, thanks @mallachari for spotting it.